content/posts/mfa.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "2-Factor Authentication!"
+excerpt: "Easily secure your Aptible accounts."
+author_name: Thomas Orozco
+author_email: thomas@aptible.com
+author_id: thomas
+posted: 2016-05-19
+section: Blog
+posts: true
+---
+We're happy to announce that two-factor authentication (2FA) is available for all users and account types in the Aptible dashboard and CLI! Multifactor authentication is a best practice that adds an additional layer of security on top of the normal username and password you use to verify your identity. You can enable it in your Aptible user settings.
```

Comment:
@chasballew perhaps we could include the link here (https://dashboard.aptible.com/settings/protected/admin)?